crates/noirc_frontend/src/parser/parser.rs

```diff
@@ -1444,6 +1479,22 @@ mod test {
         })
     }
 
+    fn parse_all_ignore_warnings<P, T>(parser: P, programs: Vec<&str>) -> Vec<T>
```

Comment:
Can you update the default `parse_all` to ignore warnings instead?

Reply:
Done